test/annexB/built-ins/escape/argument_types.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2020 Qu Xing. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-escape-string
+description: Input is a null, undefined or boolean.
+info: |
+	B.2.1.1 escape(string)
+	[...]
+    	1. Let string be ? ToString(string).
+    	
+    7.1.17 ToString(argument)
```

Comment:
I think we can drop `L11-L22`: `ToString` is pretty well-known and `L13` causes frontmatter parsing error.